Source/scrollrt.cpp

```diff
@@ -459,13 +459,13 @@ void DrawMonster(const Surface &out, Point tilePosition, Point targetBufferPosit
 /**
  * @brief Helper for rendering a specific player icon (Mana Shield or Reflect)
  */
-void DrawPlayerIconHelper(const Surface &out, int pnum, missile_graphic_id missileGraphicId, Point position, bool lighting)
+void DrawPlayerIconHelper(const Surface &out, Player &player, missile_graphic_id missileGraphicId, Point position, bool lighting)

```

Comment:
Another case where the player reference isn't used, though this does make the calling function signature nicer. Maybe have a helper like the following so this function can take a bool?

```cpp
bool Player::isLocalPlayer() const { return this == MyPlayer; }
```